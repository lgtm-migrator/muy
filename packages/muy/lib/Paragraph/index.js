import React from "react"
import Typography from "@material-ui/core/Typography"

const Paragraph = props => <Typography paragraph={true} {...props} />

export default Paragraph
