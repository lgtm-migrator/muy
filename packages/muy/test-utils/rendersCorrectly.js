import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
