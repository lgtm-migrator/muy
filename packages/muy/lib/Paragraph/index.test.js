import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import Paragraph from "."

describe("<Paragraph /> component", () => {
  it("renders correctly", () => {
    rendersCorrectly(<Paragraph />)
  })
})
