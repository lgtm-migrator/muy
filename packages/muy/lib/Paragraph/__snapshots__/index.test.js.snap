// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph /> component renders correctly 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
/>
`;
